autoload/gutentags.vim

```diff
@@ -98,7 +98,7 @@ function! gutentags#setup_gutentags() abort
     " Try and find what tags file we should manage.
     call gutentags#trace("Scanning buffer '" . bufname('%') . "' for gutentags setup...")
     try
-        let b:gutentags_root = gutentags#get_project_root(expand('%:h'))
+        let b:gutentags_root = gutentags#get_project_root(expand('%:p:h'))
```

Comment:
Actually it looks like it's OK in those cases. Calling `expand()` with an argument that starts with any of the special stuff (`%`, `#`, `<sfile>`, etc) makes it somehow not use `wildignore`. At least that's what I'm seeing here when I actually open one of my ignored file (like a `.pyc` file).